suite.go

```diff
@@ -0,0 +1,110 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import (
+	"reflect"
+	"strings"
+	"unicode"
+	"unicode/utf8"
+)
+
+// RunSuite runs all the tests defined in the given suite as a separate
+// subtest. A test is a method on the suite of the form
+//	func (T) TestXxx(*quicktest.C)
+// where Xxx does not start with a lowercase letter.
+//
+// If suite is is a pointer, the value pointed to is copied
+// before any methods are invoked on it; a new copy
+// is made for each test.
```

Comment:
This means that it is OK for tests to modify fields in suite
concurrently if desired - it's OK to call `c.Parallel()`.

?